src/components/register/story.js

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
-import { storiesOf } from '@kadira/storybook';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions'; //eslint-disable-line
 import Register from './';
 
+const onChange = () => {
+  action('register on change');
+};
+
 storiesOf('Register')
   .add('User registration', () => (
-    <Register />
+    <Register handleChange={onChange} />
```

Comment:
To get rid of the console errors when viewing the Register component and if you typed into the input boxes.